ffs/scheduler/scheduler_import.go

```diff
@@ -0,0 +1,124 @@
+package scheduler
```

Comment:
Here the logic of deal importing.
This code has some comments, so I'll add extra ones in review if make sense.
Feel free to ask.